### ISSUE TITLE
set commentstring in ftplugin

### ISSUE THREAD

**Issue by dhduvall** [nickspoons/vim-cs#66]
Vim documents `commentstring` as
> Currently only used to add markers for folding

which might explain why many filetype plugins don't set it; but I discovered that [vim-commentary](https://github.com/tpope/vim-commentary) uses it, so I think it's helpful to have set usefully. I was surprised to find it prepending `# ` to the commented-out lines, but that was probably because of some other plugin setting it globally instead of locally.

Easy workaround:
```vim
autocmd FileType cs setl commentstring=//%s
```

**Comment by nickspoons** [comment]
I'm not really sure if it's appropriate to set this, just for the sake of vim-commentary. Especially since the global vim default of `/*%s*/` is valid in C#.

**Comment by dhduvall** [comment]
Fair enough. I didn't look for other users; I just hypothesized their existence. I'll defer to you on this. (I should definitely track down what plugin set it globally, though—that seems like a bug.)

**Comment by nickspoons** [comment]
You can find out where an option was set using `:verbose`, e.g.:

```vim
:verb set commentstring?
```

**Comment by nickspoons** [comment]
The `commentstring` is now included, see #71. Both vim and neovim now include built-in commenting functionality which make use of it, and it has become much more standard to set the option in filetype plugins.
